python/cugraph/cugraph/structure/graph_classes.py

```diff
@@ -174,6 +175,12 @@ def from_cudf_edgelist(
 
             This parameter is deprecated and will be removed.
 
+        symmetrize: bool, optional (default=None)
+            If True, symmetrize the edge list for an undirected graph. Setting
+            this flag to True for a directed graph returns an error. The default
+            behavior symmetrizes the edges if the graph is undirected. If the edges
+            are already symmetric, set this flag to False.
```

Comment:
Should you also mention the restriction on edgelists with edge IDs and edge types?  Also, the way this is worded makes it seem like the user must figure out if the edgrs are symmetric or not and set the flag accordingly.  Maybe it would be better if it was worded like this:
```suggestion
            behavior symmetrizes the edges if the graph is undirected. This flag cannot be set to True if the edgelist contains edge IDs or edge Types.  If the incoming edgelist is intended for an undirected graph and it is known to be symmetric, this flag can be set to False to skip the symmetrization step for better performance.
```
Same suggestion for `from_cudf_adjlist` below.

Reply:
Got it. I reworded it as you suggested